<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="options_lbl">
     <property name="font">
      <font>
       <family>Harlow Solid Italic</family>
       <pointsize>25</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Nastavení</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="spacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="number_of_cards_layout">
     <item>
      <widget class="QLabel" name="number_of_cards_lbl">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Počet dvojic:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="number_of_cards_spb">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>11</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="spacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="main_rule_layout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Střídání hráčů po nalezení dvojice:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="main_rule_chb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <spacer name="spacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="back_btn">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Zpět</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Uložit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="spacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
